content/login-from-live-disk.md

```diff
@@ -46,7 +48,9 @@ If the command fails and says `mount: /mnt: unknown filesystem type 'crypto_LUKS
 
 ### Encrypted Disk
 
-To get access to an encrypted disk, these additional commands need to be run in order to unlock the disk.  Please use the `lsblk` command described above to determine the correct drive and partition.
+To get access to an encrypted disk, these additional commands need to be run in order to unlock the disk.  
```

Comment:
*To access an encrypted disk, run these additional commands to unlock the encrypted volume.